apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.Tests/SpannerParameterTests.cs

```diff
@@ -46,6 +48,29 @@ public void DbTypeMappgings(SpannerDbType spannerType, DbType adoType)
             Assert.Equal(spannerType, parameter.SpannerDbType);
         }
 
+        // TODO: There is no value that will default to Spanner type DATE.
+        public static IEnumerable<object[]> GetValueConversions()
+        {
+            yield return new object[] { new byte[] { 1 }, SpannerDbType.Bytes, DbType.Binary };
+            yield return new object[] { true, SpannerDbType.Bool, DbType.Boolean };
+            yield return new object[] { new DateTime(2021, 1, 13, 12, 3, 10), SpannerDbType.Timestamp, DbType.DateTime };
+            yield return new object[] { 3.14D, SpannerDbType.Float64, DbType.Double };
+            yield return new object[] { 1L, SpannerDbType.Int64, DbType.Int64 };
+            yield return new object[] { (SpannerNumeric)3.14m, SpannerDbType.Numeric, DbType.VarNumeric };
```

Comment:
The literal suffixes are case insensitive, but you've used capital letters on the ones above, so, for consistency:
```suggestion
            yield return new object[] { (SpannerNumeric)3.14M, SpannerDbType.Numeric, DbType.VarNumeric };
```

Reply:
I'd suggest the opposite, actually - I suggest we fix the suffixes above to be lower-case, as IMO that's more idiomatic C# :)

Reply:
I actually debated which was best, but got "confused" (is it lower case that I normally use, upper case?) and then I found [this](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/builtin-types/integral-numeric-types#integer-literals) (see the note there). Maybe lower case for anything except L? But I'm fine with the most idiomatic/consistent we can find ;).

Reply:
Done (everything lower case except for long). I couldn't really find anything authoritative convention for this.